test/test_overlay.py

```diff
@@ -42,8 +40,10 @@ def test_overlay_cp2k_singlepoint():
     g.project = "cp2k"
     g.run_type = "energy"
 
-    print(s.specific.cp2k.force_eval)
-    print(r.specific.cp2k.force_eval)
+    r = templates.singlepoint.overlay(s)
+
+    print("created:\n", s.specific.cp2k.force_eval)
```

Comment:
It might be a bit cleaner if this was passed to a logger.

Reply:
I agree, but I am going to delay the introduction of a logger for a bit

Reply:
Yeah, fair enough.
This might be a bit more work to set up.